catalog/USGS/USGS_NLCD_RELEASES_2019_REL_RCMAP_V5_TRENDS_YEAR.jsonnet

```diff
@@ -0,0 +1,441 @@
+        'gee:scale': 0.01,
+        'gee:units': units.percent,
+      },
+      {
+        name: 'tree_segment_slope',
+        description: 'Slope of structural breaks segment, given in units of % change/year, in each component for each year.',
+        'gee:scale': 0.01,
+        'gee:units': units.percent,
+      },
+    ],
+    annual_herbaceous_break_point: {minimum: 0, maximum: 1, 'gee:estimated_range': false},
+    bare_ground_break_point: {minimum: 0, maximum: 1, 'gee:estimated_range': false},
+    herbaceous_break_point: {minimum: 0, maximum: 1, 'gee:estimated_range': false},
+    litter_break_point: {minimum: 0, maximum: 1, 'gee:estimated_range': false},
+    sagebrush_break_point: {minimum: 0, maximum: 1, 'gee:estimated_range': false},
+    shrub_break_point: {minimum: 0, maximum: 1, 'gee:estimated_range': false},
+    non_sagebrush_shrub_break_point: {minimum: 0, maximum: 1, 'gee:estimated_range': false},
+    perennial_herbaceous_break_point: {minimum: 0, maximum: 1, 'gee:estimated_range': false},
+    tree_break_point: {minimum: 0, maximum: 1, 'gee:estimated_range': false},
+    annual_herbaceous_segment_pvalue: {minimum: 0, maximum: 100, 'gee:estimated_range': false},
+    bare_ground_segment_pvalue: {minimum: 0, maximum: 100, 'gee:estimated_range': false},
+    herbaceous_segment_pvalue: {minimum: 0, maximum: 100, 'gee:estimated_range': false},
+    litter_segment_pvalue: {minimum: 0, maximum: 100, 'gee:estimated_range': false},
+    sagebrush_segment_pvalue: {minimum: 0, maximum: 100, 'gee:estimated_range': false},
+    shrub_segment_pvalue: {minimum: 0, maximum: 100, 'gee:estimated_range': false},
+    non_sagebrush_shrub_segment_pvalue: {minimum: 0, maximum: 100, 'gee:estimated_range': false},
+    perennial_herbaceous_segment_pvalue: {minimum: 0, maximum: 100, 'gee:estimated_range': false},
+    tree_segment_pvalue: {minimum: 0, maximum: 100, 'gee:estimated_range': false},
+    annual_herbaceous_segment_slope: {minimum: -99999, maximum: 99999, 'gee:estimated_range': false},
+    bare_ground_segment_slope: {minimum: -99999, maximum: 99999, 'gee:estimated_range': false},
+    herbaceous_segment_slope: {minimum: -99999, maximum: 99999, 'gee:estimated_range': false},
+    litter_segment_slope: {minimum: -99999, maximum: 99999, 'gee:estimated_range': false},
+    sagebrush_segment_slope: {minimum: -99999, maximum: 99999, 'gee:estimated_range': false},
+    shrub_segment_slope: {minimum: -99999, maximum: 99999, 'gee:estimated_range': false},
+    non_sagebrush_shrub_segment_slope: {minimum: -99999, maximum: 99999, 'gee:estimated_range': false},
+    perennial_herbaceous_segment_slope: {minimum: -99999, maximum: 99999, 'gee:estimated_range': false},
+    tree_segment_slope: {minimum: -99999, maximum: 99999, 'gee:estimated_range': false},
+  },
+  'gee:interval': {
+   type: 'cadence',
+   unit: 'year',
+   interval: 1,

```

Comment:
Add one additional space of indentation inside 'gee:interval'

Reply:
Updated